validator/src/cli.rs

```diff
@@ -612,7 +612,16 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
             Arg::with_name("skip_poh_verify")
                 .long("skip-poh-verify")
                 .takes_value(false)
-                .help("Skip ledger verification at validator bootup"),
+                .help(
+                    "Deprecated, please use --skip-verification.\n\
+                     Skip ledger verification at validator bootup."
+                ),
+        )
+        .arg(
+            Arg::with_name("skip_verification")
```

Comment:
I'm not sure I love this new arg ... I know there is the help message but it is pretty generic compared to some of our other args

Reply:
`--skip-startup-ledger-verification`?

Reply:
Took another look to be sure, I like `--skip-startup-ledger-verification`; it is a good level of specific and terminology is consistent with other terms in argument names/help.

Reply:
:+1: Here is a renaming PR: https://github.com/solana-labs/solana/pull/30973